Azure/communication-ui-library-ios
# [Feature] added release script for cocoapod + improvement

## Purpose
- made main repo script arguments to be case insensitive
- added release script for podspec

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test
1. run the release script locally
2. verify the script by typing `git status` in the terminal and see if all changes have been applied

## What to Check
1. make sure podspec gets updated

## Other Information
<img width="475" alt="image" src="https://user-images.githubusercontent.com/109105353/212758650-bd76d603-4b79-416e-a771-4bc2cba46788.png">
<img width="575" alt="image" src="https://user-images.githubusercontent.com/109105353/212758695-a495a7ec-655e-412d-89a0-8169969ea730.png">
